Description: No threaded testsuite
 So that it compiles on all architectures.
Author: Joachim Breitner <nomeata@debian.org>

--- a/base64-bytestring.cabal
+++ b/base64-bytestring.cabal
@@ -59,7 +59,7 @@ test-suite test
   type:             exitcode-stdio-1.0
   hs-source-dirs:   tests
   main-is:          Tests.hs
-  ghc-options:      -Wall -threaded -rtsopts
+  ghc-options:      -Wall -rtsopts
   build-depends:
       base
     , base64-bytestring
